App.js

```diff
@@ -14,7 +14,7 @@ export default class App extends React.Component {
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: '#fff',
+    backgroundColor: '#000',
```

Comment:
Hey @nickzuber-alt-but-cooler I'd love to get your thoughts on a design-standpoint here?

Reply:
Me again @nickzuber-alt-but-cooler !!!!! 🎉